src/app/log-tool/export-modal/export-modal.component.html

```diff
@@ -22,14 +22,19 @@ <h3>Export Data</h3>
                     Download data from last use ({{data.modifiedDate | date}}) to be imported at a later time.
                 </p>
                 <div class="form-group">
-                    <label for="exportName"> Name your file:</label>
+                    <label for="exportName"> Name your file:&nbsp;</label>
                     <input type="text" class="form-control w-100" id="exportName" [(ngModel)]="exportName">
                 </div>
             </div>
             <div class="modal-footer justify-content-between">
                 <button class="btn btn-secondary" (click)="closeExportData()">Cancel</button>
-                <button class="btn btn-primary ml-2" [disabled]="!dataExists" (click)="buildExportJSON()">Export
-                    as .json</button>
+                <div class = "d-flex">
```

Comment:
We should simplify things for the user.
A. They probably won't be familiar with gz's, or file compression. I think we're better off not displaying any message at all, OR something similar to other long-processing areas of the app...  "Exporting to file... this may take some time depending on the size of your data" Something like that.
B. Similarly let's just have one button that says 'Export'. Our users don't really need to understand whether it's gz/json, unpacking a gz, or really using the files anywhere else (especially since you enabled them to import the same gz)

![image (12)](https://github.com/ORNL-AMO/AMO-Tools-Desktop/assets/60401912/b602b9e7-6eca-44bc-93ce-64ae23f661b0)